.travis.yml

```diff
@@ -9,6 +9,8 @@ addons:
       - texlive-latex-extra
 
 before_install:
+  - openssl aes-256-cbc -K $encrypted_e5009b71da55_key -iv $encrypted_e5009b71da55_iv
```

Comment:
I think we need to conditionally do this if it's not a PR as well to prevent https://travis-ci.org/certbot/website/builds/253435308#L371.